store/streaming/plugins/abci/v1/grpc.go

```diff
@@ -0,0 +1,148 @@
+package v1
+
+import (
+	"context"
+	"os"
+
+	"github.com/hashicorp/go-plugin"
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
```

Comment:
store cant depend on sdk could we remove this import

Reply:
The plugin needs to `UnwrapSDKContext` to get access to the `BlockHeight` and the `StreamingManager`. We need to make a compromise here if we are to house streaming under store, otherwise, we move streaming back to root. Let me know which takes precedence and I'll make the necessary changes.

Reply:
neither is really a good option imo. we have purposefully removed sdk from store and moved streaming to store. There is a way around this. We just have to figure it out

Reply:
To remove the sdk dependency I introduced a `Context` interface. This interface is implemented by `types.Context` in the SDK. With store being a standalone module, I found introducing an interface to pass context information to be a good approach here. Would like some feedback on this.

https://github.com/cosmos/cosmos-sdk/blob/31a9e9c49299bbc65f804d3bf99182da98f43c83/store/types/context.go#L8-L15

implementation in sdk:
https://github.com/cosmos/cosmos-sdk/blob/31a9e9c49299bbc65f804d3bf99182da98f43c83/types/context.go#L313-L314

use in plugin:
https://github.com/cosmos/cosmos-sdk/blob/31a9e9c49299bbc65f804d3bf99182da98f43c83/store/streaming/abci/grpc.go#L20-L26